server.py

```diff
@@ -44,28 +61,33 @@ def donation():
         # param is '' in this case
         print("Param is: %s" % err['param'])
         print("Message is: %s" % err['message'])
-    except (stripe.error.InvalidRequestError, e):
+    except stripe.error.InvalidRequestError as e:
         # Invalid parameters were supplied to Stripe's API
         print(e)
         pass
-    except (stripe.error.AuthenticationError, e):
+    except stripe.error.RateLimitError as e:
+        # Too many requests made to the API too quickly
+        print(e)
+        pass
+    except stripe.error.AuthenticationError as e:
         # Authentication with Stripe's API failed
         # (maybe you changed API keys recently)
         print(e)
         pass
-    except (stripe.error.APIConnectionError, e):
+    except stripe.error.APIConnectionError as e:
         # Network communication with Stripe failed
         print(e)
         pass
-    except (stripe.error.StripeError, e):
+    except stripe.error.StripeError as e:
+        # Display a very generic error to the user, and maybe send
+        # yourself an email
         print(e)
         pass
-    except (Exception, e):
+    except Exception as e:
         # Something else happened, completely unrelated to Stripe
         print(e)
         pass
-    # return render_template('do we need to render again?', paid=True)
-
+    return render_template('donation_successful.html', amount=amount)
 
-app.run(host='0.0.0.0',
-        port=int(os.getenv('PORT', 5000)))
+#app.run(debug=True, host='localhost', port=int(os.getenv('PORT', 5000)))
```

Comment:
If we're not doing to pass debug here (as well we shouldn't) I'd like to see this line removed and not have it in a comment. 

Or, even better I'd love this passed as a variable that we can conditionally set based on an  environment variable as well.